libs/commons/UFile.mli

```diff
@@ -66,6 +66,26 @@ val read_file : ?max_len:int -> Fpath.t -> string
 val with_open_out : Fpath.t -> ((string -> unit) * out_channel -> 'a) -> 'a
 val with_open_in : Fpath.t -> (in_channel -> 'a) -> 'a
 
+(* Create a temporary file, run the user function, and clean up.
```

Comment:
You should use UTmp.ml for those temp related functions, not UFile.ml

Reply:
yep